SampleApplications/Samples/XamarinClient/XamarinClient.Android/Resources/values/Strings.xml

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="hello">Hello World, Click Me!</string>
+	<string name="app_name">XamarinClient.Droid</string>
+</resources>
```

Comment:
for what is this used?

Reply:
deleted, it's just a sample file created by the Xamarin Visual Studio project template